OAuth: Removes send_client_credentials_via_post setting (#20044)
grafana/grafana: docs/sources/installation/upgrading.md

> #### Upgrade from binary .tar file

Reviewer: ```suggestion
#### Upgrade from binary .tar.gz file
```

Reviewer 2: These are unrelated so suggest adding them in separate PR

> If you downloaded the binary `.tar.gz` package, then you can just download and extract the new package and overwrite all your existing files. However, this might overwrite your config changes.

Reviewer: ```suggestion
If you downloaded the binary `.tar.gz` package, you can just download and extract the new package and overwrite all your existing files. However, this might overwrite your config changes.
```

Reviewer 2: These are unrelated so suggest adding them in separate PR

Reviewer: OK @marefr 👍

Each request to the GetMetricData API can include 100 queries. This means that each panel in Grafana will only issue one GetMetricData request, regardless of the number of query rows that are present in the panel. Consequently as it is no longer possible to set `HighRes` on a per query level anymore, this switch is now removed from the query editor. High resolution can still be achieved by choosing a smaller minimum period in the query editor.

The handling of multi-valued template variables in dimension values has been changed in Grafana 6.5. When a multi template variable is being used, Grafana will generate a search expression. In the GetMetricData API, expressions are limited to 1024 characters, so it might be the case that this limit is reached when a multi-valued template variable that has a lot of values is being used. If this is the case, we suggest you start using `*` wildcard as dimension value instead of a multi-valued template variable.

## Upgrading to v6.6

> The Generic OAuth setting, `send_client_credentials_via_post`, used for supporting non-compliant providers have been removed. From now on Grafana will automatically detect if credentials should be sent as part of url or body for a specific provider. The result will be remembered and used for additional OAuth requests for that provider.

Reviewer: ```suggestion
The Generic OAuth setting `send_client_credentials_via_post`, used for supporting non-compliant providers, has been removed. From now on, Grafana will automatically detect if credentials should be sent as part of the URL or request body for a specific provider. The result will be remembered and used for additional OAuth requests for that provider.
```
